--- conf/nginx.conf.orig	2007-03-22 22:44:23.000000000 -0600
+++ conf/nginx.conf	2007-03-22 23:06:48.000000000 -0600
@@ -1,12 +1,11 @@
-
-#user  nobody;
+user  nginx;
 worker_processes  1;
 
-#error_log  logs/error.log;
-#error_log  logs/error.log  notice;
-#error_log  logs/error.log  info;
+error_log   /var/log/nginx/error.log;
+#error_log  /var/log/nginx/error.log  notice;
+#error_log  /var/log/nginx/error.log  info;
 
-#pid        logs/nginx.pid;
+pid        /var/run/nginx.pid;
 
 
 events {
@@ -15,14 +14,14 @@
 
 
 http {
-    include       conf/mime.types;
+    include       /etc/nginx/mime.types;
     default_type  application/octet-stream;
 
-    #log_format  main  '$remote_addr - $remote_user [$time_local] $request '
-    #                  '"$status" $body_bytes_sent "$http_referer" '
-    #                  '"$http_user_agent" "$http_x_forwarded_for"';
+    log_format  main  '$remote_addr - $remote_user [$time_local] $request '
+                      '"$status" $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
 
-    #access_log  logs/access.log  main;
+    access_log  /var/log/nginx/access.log  main;
 
     sendfile        on;
     #tcp_nopush     on;
@@ -38,7 +37,7 @@
 
         #charset koi8-r;
 
-        #access_log  logs/host.access.log  main;
+        #access_log  /var/log/nginx/host.access.log  main;
 
         location / {
             root   html;
@@ -113,5 +112,4 @@
     #        index  index.html index.htm;
     #    }
     #}
-
 }
