--- auto/install.orig	2007-03-22 09:30:15.000000000 -0600
+++ auto/install	2007-03-22 09:40:42.000000000 -0600
@@ -19,42 +19,42 @@
 
 install:	$NGX_OBJS${ngx_dirsep}nginx${ngx_binext}	\
 		$NGX_INSTALL_PERL_MODULES
-	test -d '$NGX_PREFIX' || mkdir -p '$NGX_PREFIX'
+	test -d '$NGX_DEST_DIR$NGX_PREFIX' || mkdir -p '$NGX_DEST_DIR$NGX_PREFIX'
 
-	test -d '`dirname "$NGX_SBIN_PATH"`' \
-		|| mkdir -p '`dirname "$NGX_SBIN_PATH"`'
-	test ! -f '$NGX_SBIN_PATH' || mv '$NGX_SBIN_PATH' '$NGX_SBIN_PATH.old'
-	cp $NGX_OBJS/nginx '$NGX_SBIN_PATH'
+	test -d '`dirname "$NGX_DEST_DIR$NGX_SBIN_PATH"`' \
+		|| mkdir -p '`dirname "$NGX_DEST_DIR$NGX_SBIN_PATH"`'
+	test ! -f '$NGX_DEST_DIR$NGX_SBIN_PATH' || mv '$NGX_DEST_DIR$NGX_SBIN_PATH' '$NGX_DEST_DIR$NGX_SBIN_PATH.old'
+	cp $NGX_OBJS/nginx '$NGX_DEST_DIR$NGX_SBIN_PATH'
 
-	test -d '`dirname "$NGX_CONF_PATH"`' \
-		|| mkdir -p '`dirname "$NGX_CONF_PATH"`'
+	test -d '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`' \
+		|| mkdir -p '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`'
 
-	cp conf/koi-win '`dirname "$NGX_CONF_PATH"`'
-	cp conf/koi-utf '`dirname "$NGX_CONF_PATH"`'
-	cp conf/win-utf '`dirname "$NGX_CONF_PATH"`'
+	cp conf/koi-win '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`'
+	cp conf/koi-utf '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`'
+	cp conf/win-utf '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`'
 
-	test -f '`dirname "$NGX_CONF_PATH"`/mime.types' || \
-		cp conf/mime.types '`dirname "$NGX_CONF_PATH"`/mime.types'
-	cp conf/mime.types '`dirname "$NGX_CONF_PATH"`/mime.types.default'
+	test -f '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`/mime.types' || \
+		cp conf/mime.types '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`/mime.types'
+	cp conf/mime.types '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`/mime.types.default'
 
-	test -f '$NGX_CONF_PATH' || cp conf/nginx.conf '$NGX_CONF_PATH'
-	cp conf/nginx.conf '`dirname "$NGX_CONF_PATH"`/nginx.conf.default'
+	test -f '$NGX_DEST_DIR$NGX_CONF_PATH' || cp conf/nginx.conf '$NGX_DEST_DIR$NGX_CONF_PATH'
+	cp conf/nginx.conf '`dirname "$NGX_DEST_DIR$NGX_CONF_PATH"`/nginx.conf.default'
 
-	test -d '`dirname "$NGX_PID_PATH"`' \
-		|| mkdir -p '`dirname "$NGX_PID_PATH"`'
+	test -d '`dirname "$NGX_DEST_DIR$NGX_PID_PATH"`' \
+		|| mkdir -p '`dirname "$NGX_DEST_DIR$NGX_PID_PATH"`'
 
-	test -d '`dirname "$NGX_HTTP_LOG_PATH"`' || \
-		mkdir -p '`dirname "$NGX_HTTP_LOG_PATH"`'
+	test -d '`dirname "$NGX_DEST_DIR$NGX_HTTP_LOG_PATH"`' || \
+		mkdir -p '`dirname "$NGX_DEST_DIR$NGX_HTTP_LOG_PATH"`'
 
-	test -d '$NGX_PREFIX/html' || cp -r html '$NGX_PREFIX'
+	test -d '$NGX_DEST_DIR$NGX_PREFIX/html' || cp -r html '$NGX_DEST_DIR$NGX_PREFIX'
 END
 
 
-if test -n "$NGX_ERROR_LOG_PATH"; then
+if test -n "$NGX_DEST_DIR/$NGX_ERROR_LOG_PATH"; then
     cat << END                                                >> $NGX_MAKEFILE
 
-	test -d '`dirname "$NGX_ERROR_LOG_PATH"`' || \
-		mkdir -p '`dirname "$NGX_ERROR_LOG_PATH"`'
+	test -d '`dirname "$NGX_DEST_DIR/$NGX_ERROR_LOG_PATH"`' || \
+		mkdir -p '`dirname "$NGX_DEST_DIR/$NGX_ERROR_LOG_PATH"`'
 END
 
 fi
